samples/04_gis_analysts_data_scientists/Training a Deep Learning model, for using it in 3D basemap Solutions..ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>The <code>save()</code> function is used to the save the trained model. The saved folder will contain the ESRI model definition file, which will be required by the "Deep Learning Tools" in the 3D Basemap Solution as an input for referencing. Alternatively, <code>predict_las( )</code> can be used for inferencing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
corrected accordingly, Thanks.